spec/applications/business_finder_email_spec.rb

```diff
@@ -0,0 +1,43 @@
+feature "Default email frequency", finder_frontend: true do
+  context "within the Business Finder" do
+    let(:signup_url) { "/find-eu-exit-guidance-business/email-signup" }
+    let(:expected_option) { "daily" }
+
+    scenario "email frequency should be daily" do
+      when_i_visit signup_url
+      and_i_click_create_subscription
+      then_this_option_should_be_selected expected_option
+    end
+  end
+
+  context "within other Finders" do
+    let(:signup_url) { "/drug-device-alerts/email-signup" }
+    let(:expected_option) { "immediately" }
+
+    scenario "email frequency should be immediately" do
+      when_i_visit signup_url
+      and_i_click_create_subscription
+      then_this_option_should_be_selected expected_option
+    end
+  end
+
+private
+
+  def when_i_visit(signup_url)
+    visit "#{Plek.new.website_root}#{signup_url}"
+    puts "FINDER CURRENT URL #{current_url}"
+    puts "FINDER PAGE CONTENT #{page.text}"
```

Comment:
![image](https://user-images.githubusercontent.com/5111927/57382683-4b658e00-71a5-11e9-9bfa-6d1f68b77d88.png)

@oscarwyatt any ideas why the "/drug-device-alerts/email-signup" signup page resolves but "/find-eu-exit-guidance-business/email-signup" 404s...?

Reply:
Not sure actually, although for my test I used `visit("#{Plek.new.website_root}/find-eu-exit-guidance-business")` which does resolve. It's also worth looking in the logs of the run to check that the finder config and finder email config are published correctly - without that the 404 is correct

Reply:
In fact, you'll need to wait until my test is merged and rebase yours (or just manually add in the diff) before it'll resolve because it adds the steps to publish the business finder facets

Reply:
Ahh brilliant, thanks 👍